site/components/pages/Home/index.tsx

```diff
@@ -175,6 +180,23 @@ export const Home: NextPage<Props> = (props) => {
           </div>
         </div>
 
+        <div className={styles.ctaButtonContainer}>
+          <ButtonSecondary
+            label={t({
+              id: "shared.infinity_cta",
+              message: "DEFI FOR ORGANIZATIONS",
```

Comment:
Please use Normal case in the source string and use CSS to uppercase it `text-transform: uppercase`

```suggestion
              message: "DeFi for organizations",
```

Reply:
fixed

Reply:
The buttons werent on the mobile view on figma so I hid them but they are displayed on mobile now.